microsoft/testsuites/sysbench/sysbenchperf.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from pathlib import Path
+from typing import Any, Dict, List
+
+from assertpy.assertpy import assert_that, fail
+
+from lisa import (
+    Environment,
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+)
+from lisa.testsuite import TestResult
+from lisa.tools import Sysbench
+from lisa.tools.sysbench import SysbenchTestResult
+
+
+@TestSuiteMetadata(
+    area="sysbench",
+    category="performance",
+    description="""
+    This test suite is for executing the sysbench tests
+    """,
+)
+class SysbenchTestSuite(TestSuite):
+    @TestCaseMetadata(
+        description="""
+            Runs Sysbench test for cpu
+        """,
+        priority=3,
+    )
+    def perf_sysbench_cpu(
+        self,
+        log: Logger,
+        node: Node,
+        environment: Environment,
+        log_path: Path,
+        result: TestResult,
+        variables: Dict[str, Any],
+    ) -> None:
+        subtest_name: str = "default_sysbench_cpu_perf_test"
+        output: SysbenchTestResult = node.tools[Sysbench].run_cpu_perf(
+            subtest_name=subtest_name,
+            test_result=result,
+        )
+        failure_msg = f"Sysbench-cpu test failed with exit code {output.exit_code}"
+        assert_that(output.exit_code).described_as(failure_msg).is_equal_to(0)
+
+    @TestCaseMetadata(
+        description="""
+            Runs Sysbench test for fileio
+        """,
+        priority=3,
+    )
+    def perf_sysbench_fileio(
+        self,
+        log: Logger,
+        node: Node,
+        environment: Environment,
+        log_path: Path,
+        result: TestResult,
+        variables: Dict[str, Any],
+    ) -> None:
+        io_mode_ops = {
+            "seqwr": "write",
```

Comment:
It looks the map is easy to infer from left to right. So it doesn't need to be two parameters to input. It can input only one str list, and the value is evaluated in the tool.

Reply:
Done. Please check latest push